src/xfieldhistory.cpp

```diff
@@ -82,4 +82,14 @@ bool IsXFieldNew(const CXField& xfield, CXFieldHistoryMap* pxfieldHistory)
     return xfield.IsValid()
           && std::find(XFIELDTYPES_INIT_LIST.begin(), XFIELDTYPES_INIT_LIST.end(), xfield.xfieldType) != XFIELDTYPES_INIT_LIST.end()
           && !boost::apply_visitor(checkVisitor, xfield.xfieldValue);
-}
\ No newline at end of file
+}
+
+uint32_t CURRENT_MAXBLOCKSIZE() {
```

Comment:
These upper snake cases are easily mistaken for constants, so the function name should be camel case as well as others.